test/Core.Test/Services/StripePaymentServiceTests.cs

```diff
@@ -229,7 +235,10 @@ public async void PurchaseOrganizationAsync_Stripe_RequiresAction(SutProvider<St
     public async void PurchaseOrganizationAsync_Paypal(SutProvider<StripePaymentService> sutProvider, Organization organization, string paymentToken, TaxInfo taxInfo)
     {
         var plan = StaticStore.Plans.First(p => p.Type == PlanType.EnterpriseAnnually);
-
+        var customFields = new List<CustomerInvoiceSettingsCustomFieldOptions>()
+        {
+            new CustomerInvoiceSettingsCustomFieldOptions() { Name = "Organization", Value = organization.Name },
```

Comment:
This test should be failing, because you changed the `Name` to be `"Subscriber"`, not `"Organization"`. This means this test isn't actually testing/asserting this value. Please update this, but also make sure it's covered by the test's assertions.